site/public/css/calendar.css

```diff
@@ -0,0 +1,346 @@
+    padding: 0 0 0 0;
+    border: none;
+    box-shadow: none;
+}
+
+a.cal-gradeable-status-text:hover {
+    color: var(--text-black);
+    padding: 0 0 0 0;
+    cursor:default;
+    border: none;
+}
+
+a.cal-gradeable-status-text:focus {
+    color: var(--text-black);
+    padding: 0 0 0 0;
+    border: none;
+}
+
+/** Announcement **/
+a.cal-gradeable-status-ann {
+    color: #218ffd;  /* This color is from DS website */
+    padding: 0 0 0 0;
+    font-style: italic;
+    border: none;
+    box-shadow: none;
+}
+
+a.cal-gradeable-status-ann:hover {
+    color: #218ffd;
+    padding: 0 0 0 0;
+    font-style: italic;
+    cursor:default;
+    border: none;
+}
+
+a.cal-gradeable-status-ann:focus {
+    color: #218ffd;
+    padding: 0 0 0 0;
+    font-style: italic;
+    border: none;
+}
+
+/** Link **/
+a.cal-gradeable-status-link {
+    color: var(--external-link-blue);
+    padding: 0 0 0 0;
+    text-decoration: underline;
+    border: none;
+    box-shadow: none;
+}
+
+a.cal-gradeable-status-link:hover {
+    color: var(--external-link-hover);
+    padding: 0 0 0 0;
+    border: none;
+}
+
+a.cal-gradeable-status-link:focus {
+    color: var(--external-link-active);
+    padding: 0 0 0 0;
+    border: none;
+}
+
+/* Future homework */
+.cal-gradeable-status-0 {
+}
+
+/* Reserved: Beta homework */
+.cal-gradeable-status-1 {
+}
+
+/* OPEN TO SUBMIT */
+.cal-gradeable-status-2 {
+    background-color: var(--actionable-blue);
+}
+
+.cal-gradeable-status-2:hover {
+    background-color: var(--hover-actionable-blue);
+}
+
+.cal-gradeable-status-2:focus {
+    background-color: var(--focus-actionable-blue);
+}
+
+/* PAST DUE / LATE SUBMIT */
+.cal-gradeable-status-3 {
+    background-color: var(--danger-red);
+}
+
+.cal-gradeable-status-3:hover {
+    background-color: var(--hover-active-danger-red);
+    border: var(--hover-active-danger-red);
+}
+
+.cal-gradeable-status-3:focus {
+    background-color: var(--focus-danger-red);
+    border: var(--focus-danger-red);
+}
+
+/* CLOSED  being graded by TA/Instructor */
+a.cal-gradeable-status-4 {
+    color: var(--btn-text-black);
+    background-color: var(--btn-default-white);
+}
+
+a.cal-gradeable-status-4:hover {
+    color: var(--btn-text-black);
+    background-color: var(--btn-default-hover);
+    border: var(--btn-default-hover);
+}
+
+a.cal-gradeable-status-4:focus {
+    color: var(--btn-text-black);
+    background-color: var(--btn-default-active);
+    border: var(--btn-default-active);
+}
+
+/* GRADES AVAILABLE */
+a.cal-gradeable-status-5 {
+    color: var(--btn-text-black);
+    background-color: var(--good-green);
+}
+
+a.cal-gradeable-status-5:hover {
+    color: var(--btn-text-black);
+    background-color: var(--hover-good-green);
+    border: var(--hover-good-green);
+}
+
+a.cal-gradeable-status-5:focus {
+    color: var(--btn-text-black);
+    background-color: var(--disabled-good-green);
+    border: var(--disabled-good-green);
+}
+
+@media (max-width: 800px) {
```

Comment:
This css media query was only effecting the list of gradeables below the calendar, I thought that was part of the final page but since it got removed you can remove this for now - sorry about that

Reply:
Yes after a discussion in the meeting, the table was removed from the calendar page. But the code was still there and was disabled by a parameter. I think I would just keep those code as a hidden easter egg and in case some future feature that uses the calendar page can make use of the table, maybe